### PR TITLE
Fix bad type of result of ack function.

### PR DESCRIPTION
**Pull request by loguntsov**
Just little fix of type of resulted value.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@loguntsov Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-server&pullRequestId=2221)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-server&pullRequestId=2221) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@loguntsov Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-server&pullRequestId=2221)!

File: src/rabbit_variable_queue.erl [hunk 1]
```diff
@@ -710,7 +710,7 @@ ack([], State) ->
 ack([SeqId], State) ->
     case remove_pending_ack(true, SeqId, State) of
         {none, _} ->
-            State;
+            { [], State};
```
Comment (lukebakken) [inline src/rabbit_variable_queue.erl]: Thanks for finding this. Could you remove the leading whitespace character before `[]`? Thanks.

**Comment by michaelklishin** [comment]
Thank you. How can we reproduce the issue this PR addresses, though?

File: src/rabbit_variable_queue.erl (hunk 1 above)
Comment (loguntsov) [reply src/rabbit_variable_queue.erl]: I have done

**Comment by loguntsov** [comment]
No way. I'm just working with this code to get new type of queue.

**Comment by michaelklishin** [comment]
Fair enough, thank you for contributing! We will QA it as is.

**Comment by michaelklishin** [comment]
Backported to `v3.8.x` and `v3.7.x`.

**Pull request merged**